### PR TITLE
Devo SDK updated.

### PR DESCRIPTION
**Pull request by manas-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
1. Python-Sdk was used instead of the outdated devodsconnector to retrieve the queries for the following methods:
- ***parallel_query_helper***
- ***fetch_incidents***
- ***devo-run-query***
- ***devo-get-alerts***
- ***devo-multi-table-query***
2. Introduced the new parameter **ip_as_string** in **devo-run-query**.

## Description
Upgraded the integration from old ***devodsconnector*** to new ***devo-sdk***.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by manas-metron** [comment]
Hello, @anas-yousef There are issues that we are having with the Docker image. The **devo-sdk** version that was previously included in the Docker image produced by XSOAR is out of date, and our newly added features necessitate the use of the most recent version.Would you kindly help us with this situation?

**Comment by anas-yousef** [comment]
Hey @manas-metron, of course, I will try creating a custom docker image with the required version for **devo-sdk** (5.2.0), and I will send it to you so you could try it out.

**Comment by manas-metron** [comment]
Hello, @anas-yousef I appreciate your understanding. Once you've created a fresh custom image with the devo-sdk 5.2.0, could you kindly message me?

**Comment by anas-yousef** [comment]
Hey @manas-metron, can you configure the following docker image, **devdemisto/devo:1.0.0.93055**, in the yml file, and try running the integration with your changes?
This docker image is using devo-sdk 5.2.0.

**Comment by anas-yousef** [comment]
> Hello, @anas-yousef Would you kindly include the _pandas_ dependent to the requirements.txt file as well?

Sure thing, here you go:
devdemisto/devo:1.0.0.93057

**Comment by manas-metron** [comment]
Hello, @anas-yousef I thankful for your support. I'm able to successfully execute the integration with the updated modifications using the provided custom Docker image.

**Comment by anas-yousef** [comment]
@manas-metron Thank you for the update. I will handle the release of the docker image and update you.
Once that is done, we will conduct a small demo to check that everything is working as expected.
The process might be slightly delayed due to the upcoming holiday, I apologize in advance. The holiday will be over by April 30th.

**Comment by anas-yousef** [comment]
@manas-metron The new docker image is **demisto/devo:1.0.0.93142**, please update the yaml file to include it.

**Comment by manas-metron** [comment]
> @manas-metron The new docker image is **demisto/devo:1.0.0.93142**, please update the yaml file to include it. Can you update me via the [DFIR](https://start.paloaltonetworks.com/join-our-slack-community) slack channel regarding your availability of conducting a demo?

Hi @anas-yousef, I'll update the **demisto/devo:1.0.0.93142** Docker image in the yml file.

**Comment by manas-metron** [comment]
> @manas-metron Thank you for the update. I will handle the release of the docker image and update you. Once that is done, we will conduct a small demo to check that everything is working as expected. The process might be slightly delayed due to the upcoming holiday, I apologize in advance. The holiday will be over by April 30th.

Thanks for the advance notice.

**Comment by anas-yousef** [comment]
> > @manas-metron The new docker image is **demisto/devo:1.0.0.93142**, please update the yaml file to include it. Can you update me via the [DFIR](https://start.paloaltonetworks.com/join-our-slack-community) slack channel regarding your availability of conducting a demo?
> 
> Hi @anas-yousef, I'll update the **demisto/devo:1.0.0.93142** Docker image in the yml file. My working hours are from 10 a.m. to 6 p.m., and I'm from India.

@manas-metron Can you register to our [DFIR](https://start.paloaltonetworks.com/join-our-slack-community) slack channel, so we can decide on a time and day to conduct the demo?

**Comment by manas-metron** [comment]
> > > @manas-metron The new docker image is **demisto/devo:1.0.0.93142**, please update the yaml file to include it. Can you update me via the [DFIR](https://start.paloaltonetworks.com/join-our-slack-community) slack channel regarding your availability of conducting a demo?
> > 
> > 
> > Hi @anas-yousef, I'll update the **demisto/devo:1.0.0.93142** Docker image in the yml file. My working hours are from 10 a.m. to 6 p.m., and I'm from India.
> 
> @manas-metron Can you register to our [DFIR](https://start.paloaltonetworks.com/join-our-slack-community) slack channel, so we can decide on a time and day to conduct the demo?

Yes, I already registered for DFIR and received the Slack link, but the link is not working.

File: Packs/Devo/Integrations/Devo_v2/Devo_v2.py [hunk 1]
```diff
@@ -1027,6 +1076,8 @@ def main():
             if items_per_page <= 0:
                 raise ValueError("items_per_page should be a positive non-zero value.")
             total = 0
+            # Validate the ip_as_string param value 
+            argToBoolean(demisto.args().get("ip_as_string", IP_AS_STRING))
```
Comment (anas-yousef) [inline Packs/Devo/Integrations/Devo_v2/Devo_v2.py:1080]:
```suggestion
            ip_as_string = argToBoolean(demisto.args().get("ip_as_string", IP_AS_STRING))
```

**Comment by manas-metron** [comment]
> > > @manas-metron The new docker image is **demisto/devo:1.0.0.93142**, please update the yaml file to include it. Can you update me via the [DFIR](https://start.paloaltonetworks.com/join-our-slack-community) slack channel regarding your availability of conducting a demo?
> > 
> > 
> > Hi @anas-yousef, I'll update the **demisto/devo:1.0.0.93142** Docker image in the yml file. My working hours are from 10 a.m. to 6 p.m., and I'm from India.
> 
> @manas-metron Can you register to our [DFIR](https://start.paloaltonetworks.com/join-our-slack-community) slack channel, so we can decide on a time and day to conduct the demo?

Hi @anas-yousef I've joined the DFIR Slack channel, and I've pinned you personally on Slack. Could you please confirm it once?

**Comment by anas-yousef** [comment]
@manas-metron I saw it and messaged you back :)

**Comment by anas-yousef** [comment]
@edik24 @manas-metron
A demo was set up on April 2nd.

**Comment by manas-metron** [comment]
> @edik24 @manas-metron A demo was set up on April 2nd.

Thank you @anas-yousef.

**Review by anas-yousef — APPROVED** [review]

**Pull request merged**